toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Docs used to create this script:
+# https://preshing.com/20141119/how-to-build-a-gcc-cross-compiler/
+# https://mdeva2.home.blog/2019/07/08/building-gcc-as-a-cross-compiler-for-raspberry-pi/
+# http://jiadongsun.cc/2019/09/03/Cross_Compile_Gcc/#6-build-glibc
+
+# ====== Prerequisites ======
+# You may need to run "chmod +x <path to this script>" prior to running it to give it execute permissions
+#
+# The following packages need to be installed before running this script
+# For Ubuntu/Mariner:
+# gawk rsync bison
+
+# For Mariner, these additional packages need to be installed:
+# make patch texinfo diffutils
+# ====================
+
+
+installDir="/opt/cross"
+sysroot="${installDir}/aarch64-mariner-linux-gnu/sysroot"
+buildDir="$HOME/cross"
+scriptDir="$( cd "$( dirname "$0" )" && pwd )"
+
+sudo rm -rf ${buildDir}
+sudo rm -rf ${installDir}
+sudo rm -rf ${sysroot}
+
+mkdir -p ${buildDir}
+mkdir -p ${installDir}
+mkdir -p "${sysroot}/lib"
+
+cd ${buildDir}
+
+# Download source tarballs
+wget http://ftp.gnu.org/gnu/binutils/binutils-2.32.tar.xz
```

Comment:
We cannot download content from the internet.  The sources for these should be in the Mariner blob store.

Also, if this is really needed, each file should be sha256sum checked.